src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="header">
+    <div class="header-content container">
+        <div class="header-logo-label">
+            <div class="header-logo"><img alt={{imgText}} src="{{imageUrl}}/logo.svg"></div>
```

Comment:
```suggestion
            <div class="header-logo"><img [alt]="imgText" [src]="`${imageUrl}/logo.svg`"></div>
```

Reply:
vaia it worked this way 
_src="`{imageUrl}/searchIcon.svg`"_

Reply:
so i have create the full url in .ts and used it as a variable in html.